Week01/Problem04/YWang_04.py

```diff
@@ -0,0 +1,28 @@
+# Problem 4
+# Palindrome Product
+
+# function for testing palindrome
+def isPalindrome(n):
+  # convert number to string
+  num2str = str(n)
+  # init an empty string
+  reverseStr = ""
+  # reserse input string
+  for i in range(len(num2str)-1,-1,-1):
+    reverseStr += num2str[i]
+  return num2str == reverseStr
```

Comment:
This is a good function, but there is actually a more "pythonic" way to reverse a string (or a list!) for your code it would just be `num2str = num2str[::-1]`